string in_name
---
string out_greeting
